### PR TITLE
[NOTASK]: Updated  button to increase size

### PR DESCRIPTION
**Pull request by FevenSeyfu**
Hi while looking at the Ready to Start component  I noticed the button to be smaller than the design, hence I have made a minor change to the button component style,

- increased the font size
-  changes the horizontal padding to match the design

### CODE REVIEW

**Comment by joanwolk** [comment]
Before: alignment of button and text is weird, and button text looks off-center vertically because it's in all lowercase
Screenshot 2024-07-24 at 11 26 11 AM

After: alignment of button and text is neat, and button text vertical centering is fixed
Screenshot 2024-07-24 at 11 28 57 AM

**Comment by joanwolk** [comment]
Note that my changes are only to the "contact us" button, and not to buttons generally, because one issue was just observed when checking all uses of buttons, and the other issue arises from having all lowercase text, which is only the case in this specific button.

**Comment by FevenSeyfu** [comment]
@joanwolk After your comment I noticed I had capitalized the submit button previously which should have been all small cases on the design changed that, I have also changed the padding on all sides for the button to match the design as it is common for both buttons.

image
image

**Comment by joanwolk** [comment]
Hm. The button doesn't actually navigate to the contact page. That might be a fix for a different ticket, though.

File: src/components/ReadyToStart/ReadyToStart.module.css [hunk 1]
```diff
@@ -18,6 +18,11 @@
   margin-block: 85px;
 }
 
+.mainContent .contactLink {
+  padding-top: 8px;
+  padding-bottom: 12px;
+}
```
Comment (joanwolk) [inline src/components/ReadyToStart/ReadyToStart.module.css]: Should be able to remove this after making the Button component CSS change

Comment (FevenSeyfu) [reply src/components/ReadyToStart/ReadyToStart.module.css]: I have removed the class and style from the component. The button should now be aligned properly for both. But I have left the button action/ routing for a separate ticket.

Comment (keziabaidoo) [reply src/components/ReadyToStart/ReadyToStart.module.css]: Hello @FevenSeyfu and @joanwolk,

I have reviewed the changes made by Joan and observed that the "ready-to-get-started" element aligns well with the button in the desktop view. However, the problem still exists in the mobile view as the image is overlapping the text. I'm not sure if it has been fixed.

**Review by keziabaidoo — APPROVED** [review]

File: src/components/ReadyToStart/ReadyToStart.module.css (hunk 1 above)
Comment (keziabaidoo) [reply src/components/ReadyToStart/ReadyToStart.module.css]: Hello @joanwolk, please could you review the PR gain?

**Review by joanwolk — APPROVED** [review]

File: src/components/ReadyToStart/ReadyToStart.module.css (hunk 1 above)
Comment (joanwolk) [reply src/components/ReadyToStart/ReadyToStart.module.css]: @keziabaidoo can you please take a screenshot of what you're seeing? I don't see any overlap at any viewport width. Here are several screenshots from what I see in Firefox.

Screenshot 2024-07-24 at 11 41 10 PM

Screenshot 2024-07-24 at 11 41 48 PM

Screenshot 2024-07-24 at 11 41 59 PM

Screenshot 2024-07-24 at 11 42 24 PM

**Pull request merged**